package.json

```diff
@@ -7,7 +7,9 @@
     "Mark <mark@remarkablemark.org>"
   ],
   "scripts": {
-    "start": "nodemon index.js"
+    "client": "tsc --watch",
```

Comment:
How do you feel about naming this script to `build:client`?

Reply:
6d77aa0